src/components/CategoricalDisplay.jsx

```diff
@@ -51,6 +51,7 @@ const CategoricalDisplay = ({
         >
             {renderLeft(item)}
             <div className="right">
+                {(item.dataObject && item.dataObject.exchange_is_open == 0 )?<span className="closed-market">{t.translate("CLOSED")}</span>:''}
```

Comment:
Ha. I forgot that `CLOSED` will need to be translated. Good thing you changed it to a span.